.github/CODEOWNERS

```diff
@@ -0,0 +1,19 @@
+# This is a comment.
```

Comment:
why this file is coming as a new commit from your branch?

Aren't we following the rebase rather than merging?

Reply:
I haven't added this line